PowerEditor/installer/nsisInclude/GetVersion.nsi

```diff
@@ -0,0 +1,70 @@
+!include "FileFunc.nsh"
+!include "logiclib.nsh"
+
+OutFile "GetVersion.exe"
+SetCompressor /SOLID lzma 		; Compression technique
+RequestExecutionLevel user 		; This is to make output exe run as invoker (Admin privileges not required)
+!define EXEPATH "..\bin\Notepad++.exe"	; Absolute path
+
+
+Section "Main"
+	SetAutoClose true	; it simply writes version info in the text. So close immediatly after writing version info
+
+    DetailPrint "Getting version from ${EXEPATH}"
+    GetDllVersion "${EXEPATH}" $R0 $R1 	; Two values were read during compilation
```

Comment:
I get that the two $Rx variables/registers will contain values that are read from Notepad++ (i.e. ${EXEPATH}) but it's not clear what's the meaning of these values:
- ProductVersion and FileVersion?
- major and minor version?
- ...